handler_success.go

```diff
@@ -332,6 +338,21 @@ func (s *SuccessHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) *http
 	return nil
 }
 
+func stripListenPath(r *http.Request, listenPath string) error {
+	tmp := new(mux.Route).PathPrefix(listenPath)
```

Comment:
Not quite sure if understand how it works, can you add a comment?
Also, I feel like Regexp can be compiled only once, and cached somewhere in API def.